### PR TITLE
Adding dashboard layout

### LINKED ISSUE #47: Dashboard Layout

### PR DESCRIPTION
**Pull request by Ibtisam-Hemmo**
image

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+    //     <DashboardLayout />
+    //     <Routes>
+    //       <Route path="/" element={<Overview />}></Route>
+    //       <Route path="/admin/pharmacies" element={<Pharmacies />}></Route>
+    //       <Route path="/admin/products" element={<Products />}></Route>
+    //       <Route path="/admin/requests" element={<Requests />}></Route>
+    //       <Route path="/admin/applications" element={<Applications />}></Route>
+    //     </Routes>
+    //   </Router>
+    // </>
```
Comment (Raghad-Mezied) [inline client/src/App.tsx]: remove commented line

File: client/src/App.tsx [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+  Requests,
+  Applications,
+  DashboardLayout,
+} from "./pages";
+import "./App.css";
 
+const App = () => {
+  const router = createBrowserRouter([
+    {
+      path: "/",
```
Comment (Raghad-Mezied) [inline client/src/App.tsx]: add the admin in the main endpoint, and delete it from the the nested

File: client/src/components/NavBar.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import IconButton from "@mui/material/IconButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import AppBar from "@mui/material/AppBar";
+import AccountCircleIcon from "@mui/icons-material/AccountCircle";
+import NotificationsIcon from "@mui/icons-material/Notifications";
```
Comment (Raghad-Mezied) [inline client/src/components/NavBar.tsx]: you can import them { IconButton, MenuIcon, Toolbar} from "@mui/material"

File: client/src/components/drawer.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      <Link to="/admin/overview">
+        <ListItem>
+          <ListItemButton>
+            <ListItemIcon>
+              <DialpadIcon />
+            </ListItemIcon>
+            <ListItemText>Overview</ListItemText>
+          </ListItemButton>
+        </ListItem>
+      </Link>
```
Comment (Raghad-Mezied) [inline client/src/components/drawer.tsx]: Why to duplicate this part, create an array and loop over it

File: client/src/components/NavBar.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        sx={{ mr: 2, display: { sm: "none" } }}
+      >
+        <MenuIcon color="disabled" fontSize="large" />
+      </IconButton>
+      <Typography variant="h6" noWrap component="div">
+        Dashboard
+      </Typography>
+      <Box sx={{ display: "flex", gap: "13px", color: "grey !important" }}>
+        <NotificationsIcon fontSize="large" />
+        <AccountCircleIcon fontSize="large" />
```
Comment (Raghad-Mezied) [inline client/src/components/NavBar.tsx]: Change this to a drop menu, add an option for logout, and make it works

File: client/src/components/drawer.tsx [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+import Divider from "@mui/material/Divider";
+import { Link } from "react-router-dom";
+import DialpadIcon from "@mui/icons-material/Dialpad";
+import AttachFileIcon from "@mui/icons-material/AttachFile";
+import CategoryIcon from "@mui/icons-material/Category";
+import RequestQuoteIcon from "@mui/icons-material/RequestQuote";
+import GridViewIcon from "@mui/icons-material/GridView";
+import logo from "../assets/logo.png";
```
Comment (Raghad-Mezied) [inline client/src/components/drawer.tsx]: import as above

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

**Review by Raghad-Mezied — DISMISSED** [review]

**Review by Raghad-Mezied — APPROVED** [review]

**Review by Nader-SH — APPROVED** [review]

**Review by AhmedAbuSharar — APPROVED** [review]

**Pull request merged**